csharp/ql/test/library-tests/generics/Generics.ql

```diff
@@ -0,0 +1,252 @@
+import csharp
+
+query predicate test1(UnboundGenericDelegateType d) {
```

Comment:
I appreciate that these names were after the original test files, but do you think we can give better names to these tests predicates?

Reply:
I started doing that initially, but many of the tests are extremely ad-hoc, and hard to give good names for.